core/peer/config.go

```diff
@@ -135,6 +135,7 @@ func GetLocalAddress() (string, error) {
 	return localAddress, localAddressError
 }
 
+// GetPeerEndpoint returns peerEndpoint from cached configuration 
```

Comment:
There's an extra space at the end on line 138 which is causing goimports to fail in CI